clients/snowflake/snowflake.go

```diff
@@ -97,17 +98,21 @@ func (s *Store) reestablishConnection(ctx context.Context) {
 }
 
 func LoadSnowflake(ctx context.Context, _store *db.Store) *Store {
+	cfg := config.FromContext(ctx).Config
+
 	if _store != nil {
 		// Used for tests.
 		return &Store{
-			testDB:    true,
-			Store:     *_store,
-			configMap: &types.DwhToTablesConfigMap{},
+			testDB:            true,
+			uppercaseEscNames: cfg.SharedDestinationConfig.UppercaseEscapedNames,
+			configMap:         &types.DwhToTablesConfigMap{},
+			Store:             *_store,
```

Comment:
Do you want to add a line of whitespace here between the private and public variables like in the other store initializers?